crest/Assets/Crest/Crest/Scripts/Collision/QueryBase.cs

```diff
@@ -25,7 +25,8 @@ public abstract class QueryBase
         protected abstract string QueryShaderName { get; }
         protected abstract string QueryKernelName { get; }
 
-        const int s_maxRequests = 4;
+		// 4 was enough for a long time, but Linux setups seems to demand 7
```

Comment:
Comment indentation whitespace is tabs when file is spaces.

Reply:
Thanks